cloudberry/zion/src/main/scala/edu/uci/ics/cloudberry/zion/model/schema/Query.scala

```diff
@@ -15,15 +15,23 @@ trait IWriteQuery extends IQuery {
 
 }
 
-case class QueryExeOption(sliceMills: Int,
-                          continueSeconds: Int)
+case class JsonRequestOption(sliceMills: Int,
+                             continueSeconds: Int)
 
-object QueryExeOption {
-  val NoSliceNoContinue = QueryExeOption(-1, -1)
+object JsonRequestOption {
+  val NoSliceNoContinue = JsonRequestOption(-1, -1)
   val TagSliceMillis = "sliceMillis"
   val TagContinueSeconds = "continueSeconds"
 }
 
+case class QueryExeOption(sliceMills: Int,
```

Comment:
Now I confused, why do we need this two almost identical options?

Reply:
Because the `limit` is in `select` field somewhere in the query, and all other options are in the `option` field. We need a case class to parse the `option` field and we also need an object that contains all options including `limit`. That is the intuitive.